### ISSUE TITLE
Add user-facing deprecation, release, and Python support policies

### ISSUE THREAD

**Issue by Cadair** [sunpy/sunpy#7661]
### Describe the issue

Currently it's only in the dev guide: https://docs.sunpy.org/en/latest/dev_guide/contents/public_api.html#deprecation-policy-and-breaking-changes

### Solution

Create a topic guide on the topic.
See https://github.com/PlasmaPy/PlasmaPy/issues/2726 for a great example on how the page should be laid out.

**Comment by wtbarnes** [comment]
I agree. It can help users prepare for things being removed and explain why they are getting these deprecation warnings and what they mean. 

I wonder where the best place for this would be?

**Comment by nabobalis** [comment]
Topic guide?

**Comment by Cadair** [comment]
Also as @namurphy says in that PlasmaPy issue linked above, we should document our release schedule, version numbering system, and Python support policy at the same time. I have no idea where lol

**Comment by nabobalis** [comment]
Nick is right and topic guide it is.

**Comment by nabobalis** [comment]
We don't need to update a SEP do we?
